### ISSUE TITLE
Error: Invalid JSON RPC response on dao apps --all command

### ISSUE THREAD

**Issue by macor161** [aragon/aragon-cli#1577]
## 🐛 Bug Report

The `dao apps -all` command returns the following error when run on mainnet:

```
Invalid JSON RPC response: "<html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx/1.17.6</center>\r\n</body>\r\n</html>\r\n"
```

Using the infura endpoint doesn't seem to solve the problem.

**Comment by sohkai** [comment]
> Using the infura endpoint doesn't seem to solve the problem.

This suggests we may not be piping the `wsRpc` option to this command (or we didn't test using the latest build with it).

**Comment by macor161** [comment]
Indeed the wsRpc provider wasn't used to fetch permissionless apps :)

**Comment by macor161** [comment]
Closed by #1578
